### ISSUE TITLE
Export sceanrios filtered by tags is not correct

### ISSUE THREAD

**Issue by tikolakin** [hiptest/hiptest-publisher#61]
Hello hitest team
could you please have a look at the following attempt to export scenarios

you might know that we have tags in hiptest like `JIRA: LK-6226`
however, it makes difficulties in exporting scenarios with such tags
have a look, please 

```
15:00:27 $ ./hiptest --token lk --filter-on-tags JIRA: LK-6226, automated --no-uids --only tests
Running Hiptest-publisher 1.2.9 with:
 - __cli_args: #<Set: {:verbose, :uids, :language, :with_folders, :push_format, :token, :filter_on_tags, :only, :output_directory}>
 - __config_args: #<Set: {}>
 - language: "behat"
 - framework: ""
 - output_directory: "tests/func/ui/"
 - overriden_templates: ""
 - test_run_id: ""
 - test_run_name: ""
 - tests_only: false
 - actionwords_only: false
 - actionwords_signature: false
 - actionwords_diff: false
 - aw_deleted: false
 - aw_created: false
 - aw_renamed: false
 - aw_signature_changed: false
 - aw_definition_changed: false
 - with_folders: true
 - split_scenarios: false
 - leafless_export: false
 - site: "https://hiptest.net"
 - push: ""
 - push_format: "junit"
 - sort: "order"
 - uids: false
 - keep_filenames: false
 - keep_foldernames: false
 - filter_on_scenario_ids: ""
 - filter_on_folder_ids: ""
 - filter_on_scenario_name: ""
 - filter_on_folder_name: ""
 - filter_on_tags: "JIRA:"
 - filter_on_status: ""
 - check_version: false
 - verbose: true
 - token: "token"
 - only: "features"
```

even

```
15:00:49 $ ./hiptest --token lk --filter-on-tags="JIRA: LK-6226, automated" --no-uids --only tests
Running Hiptest-publisher 1.2.9 with:
 - __cli_args: #<Set: {:verbose, :uids, :language, :with_folders, :push_format, :token, :filter_on_tags, :only, :output_directory}>
 - __config_args: #<Set: {}>
 - language: "behat"
 - framework: ""
 - output_directory: "tests/func/ui/"
 - overriden_templates: ""
 - test_run_id: ""
 - test_run_name: ""
 - tests_only: false
 - actionwords_only: false
 - actionwords_signature: false
 - actionwords_diff: false
 - aw_deleted: false
 - aw_created: false
 - aw_renamed: false
 - aw_signature_changed: false
 - aw_definition_changed: false
 - with_folders: true
 - split_scenarios: false
 - leafless_export: false
 - site: "https://hiptest.net"
 - push: ""
 - push_format: "junit"
 - sort: "order"
 - uids: false
 - keep_filenames: false
 - keep_foldernames: false
 - filter_on_scenario_ids: ""
 - filter_on_folder_ids: ""
 - filter_on_scenario_name: ""
 - filter_on_folder_name: ""
 - filter_on_tags: "JIRA:"
 - filter_on_status: ""
 - check_version: false
 - verbose: true
 - token: "token"
 - only: "features"
```
could you pease help to sort it out and allow us to pull scenarios with tags having spaces?
thanks

**Comment by vincent-psarga** [comment]
Hi Tiko,

The problem is that you should not let the space after the colon. I'll update the publisher so it automatically strips that out, that should be easier to use this way :)

**Comment by tikolakin** [comment]
Thanks, I would try, but I also like your suggestion

**Comment by vincent-psarga** [comment]
That should be fixed in 1.4.0 :)

**Comment by tikolakin** [comment]
thanks, that is nice 👍
